### PR TITLE
added shutdown notice

### PR DESCRIPTION
**Pull request by hanbyul-here**
- added shutdown notice
- @louh can I ask you to review, merge and release?

### CODE REVIEW

**Comment by louh** [comment]
@hanbyul-here good idea. I'd like to suggest that we do this warning where the other warnings are made: https://github.com/mapzen/leaflet-geocoder/blob/master/src/core.js#L93 - and check to see if the `url` option is either not provided or contains a search.mapzen.com hostname.

It might also be a good idea to include a link to look for alternatives at the Pelias project: http://pelias.io/

File: src/core.js
Comment (louh) [inline src/core.js]: This will fail if `options` or `options.url` is undefined.

The other warnings will check to make sure `options` is defined (e.g. someone has actually specified options). It should also throw the warning if `options.url` isn't set, because it would then default to Mapzen Search.

**Review by louh — CHANGES_REQUESTED** [review]
See comment

File: src/core.js
Comment (hanbyul-here) [reply src/core.js]: do you mind if I do check against `this.options.url` directly  after default options were merged with users'? It seems to make more sense to check the final url set for this case.

**Comment by louh** [comment]
That's also fine, I think!

**Comment by hanbyul-here** [comment]
wow @coveralls  calm down.

**Comment by louh** [comment]
@hanbyul-here Looks great. I made this copy-edit change to the warning:

```
      console.warn('Mapzen is shutting down its services including Search. Read more at https://mapzen.com/blog/shutdown. To learn more about Pelias, the open-source geocoder that powers Mapzen Search, and the Pelias team’s plan for the future, please visit http://pelias.io.');
```

BUT -- I no longer have push access to this repository. So I can't actually do any maintainance tasks, regrettably.

I left a checklist for publishing here: https://github.com/mapzen/leaflet-geocoder/blob/master/CONTRIBUTING.md#versioning-and-publishing

Not everything is relevant (e.g. updating dependencies and packages), but things like updating the version number and version numbers in demos should be done. I would recommend bumping this to a new minor version (so v1.10.0).

**Comment by louh** [comment]
I'm happy to re-take over the maintainance duties on this repository, but I'm not sure if it's worthwhile to restore my access to this or if it's better if I work on it in a fork. I _think_ I still have access to the npm package, actually.

cc @dianashk

**Comment by hanbyul-here** [comment]
🤔 How do you think about moving this repo (or forking) to [nextzen github](https://github.com/nextzen) and granting you access to that repo on nextzen? I am planning to do this for lrm-mapzen.

**Comment by hanbyul-here** [comment]
Also thanks for copy editing! 🤗

**Comment by louh** [comment]
@hanbyul-here Do it and let me know!

**Comment by hanbyul-here** [comment]
well moving this repo to nextzen (and receiving commit permission) seems to cost :( I will merge this PR first we can talk more about where to moe this repo later.

**Pull request merged**
